scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -549,11 +549,19 @@ <h4 class="modal-title" id="filterNameModalLabel">Filter display name</h4>
 {% endmacro %}
 
 {% macro cell_rank(variant, institute, case, form, manual_rank_options) %}
+{% if variant.category == "sv" %}
+  <a class="variants-row-item flex-small layout"
+      href="{{ url_for('variant.sv_variant', institute_id=institute._id,
+                       case_name=case.display_name, variant_id=variant._id) }}">
+     {{ variant.variant_rank }}&nbsp;
```

Comment:
Nice! If this line is in both if and else then it could be moved outside the conditions?

Reply:
And check if the link to the variant page is also in cell_rank macro

Reply:
Pardon, what did you mean by 
>And check if the link to the variant page is also in cell_rank macro